roofit/roofitcore/src/RooAbsCollection.cxx

```diff
@@ -332,24 +347,22 @@ RooAbsCollection &RooAbsCollection::operator=(const RooAbsCollection& other)
     elem->copyCache(theirs) ;
     elem->setAttribute("Constant",theirs->isConstant()) ;
   }
-  return *this;
+  return ;
 }
 
 
-
 ////////////////////////////////////////////////////////////////////////////////
-/// The assignment operator sets the value of any argument in our set
-/// that also appears in the other set.
+/// Sets the value of any argument in our set that also appears in the other set.
 
-RooAbsCollection &RooAbsCollection::assignValueOnly(const RooAbsCollection& other, Bool_t oneSafe)
+void RooAbsCollection::assignValueOnly(const RooAbsCollection& other, bool oneSafe)
```

Comment:
This might break existing code. Maybe OK, but be prepared for angry users. At least add a sentence in the release notes.